src/app/application.cpp

```diff
@@ -233,6 +233,7 @@ Application::Application(int &argc, char **argv)
     , m_storeFileLoggerAge(FILELOGGER_SETTINGS_KEY(u"Age"_qs))
     , m_storeFileLoggerAgeType(FILELOGGER_SETTINGS_KEY(u"AgeType"_qs))
     , m_storeFileLoggerPath(FILELOGGER_SETTINGS_KEY(u"Path"_qs))
+    , m_storeFileLoggerCompressed(FILELOGGER_SETTINGS_KEY(u"Compressed"_qs))
```

Comment:
IIRC, you're only compressing backups, right? Then I would call it "CompressBackups". And `m_storeFileLoggerCompressBackups` accordingly.